scripts/docker/debian.8/Dockerfile

```diff
@@ -7,60 +7,40 @@
 FROM debian:jessie
 
 # Misc Dependencies for build
-RUN apt-get update && apt-get -qqy install curl unzip gettext sudo
+RUN apt-get update && \
+    apt-get -qqy install \
+        curl \
+        unzip \
+        gettext \
+        sudo && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
 
 # This could become a "microsoft/coreclr" image, since it just installs the dependencies for CoreCLR (and stdlib)
-RUN echo "deb http://llvm.org/apt/jessie/ llvm-toolchain-jessie-3.6 main"  | tee /etc/apt/sources.list.d/llvm.list && \
-    curl http://llvm.org/apt/llvm-snapshot.gpg.key | apt-key add - && \
```

Comment:
It's still present in the Ununtu 14.04 image.  I verified that Debian and 16.04 build cleanly still without it.  When we go to do debs on Debian we will need to figure something out.